java/code/src/com/redhat/rhn/frontend/xmlrpc/sync/iss/SyncHandler.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ */
+
+package com.redhat.rhn.frontend.xmlrpc.sync.iss;
```

Comment:
This can also be added to com.suse.manager.xmlrpc.... as we have the manager class also under com.suse.

Reply:
I didn't notice that package existed... I will move it